compose/ui/ui/src/webCommonW3C/kotlin/androidx/compose/ui/window/ComposeWindow.web.kt

```diff
@@ -586,6 +590,7 @@ private const val defaultCanvasElementId = "ComposeTarget"
  * This can be turned off by setting [applyDefaultStyles] to false.
  */
 @ExperimentalComposeUiApi
+@Deprecated("CanvasBasedWindow doesn't support HTML interop via WebElementView API. Use ComposeViewport API instead")
```

Comment:
Plase add it to release notes (probably to "Migration Notes", but it might be a part of interop feature description

Reply:
So it's better to update the Release notes in Interop Feature PR?

Reply:
up to you